### PR TITLE
refactor(migrate): Simplify engine API of evaluateDataLoss in Migrate

### PR DESCRIPTION
**Pull request by tomhoule**
Since we only care about whether the migration is empty or not, we can
simplify implementation in the engine by only returning the length of
the migration. This PR adjusts the typescript engine response type and
how the value is used to reflect that change.

Companion engines PR: https://github.com/prisma/prisma-engines/pull/2005

### CODE REVIEW

File: src/packages/migrate/src/commands/MigrateDev.ts [hunk 1]
```diff
@@ -306,7 +302,7 @@ ${chalk.green('Your database is now in sync with your schema.')}`,
       dbType,
       dbName,
       dbLocation,
-    }: { schemaWord?; dbType?; dbName?; dbLocation? },
+    }: { schemaWord?; dbType?; dbName?; dbLocation?},
```
Comment (tomhoule) [inline src/packages/migrate/src/commands/MigrateDev.ts:305]: prettier did that

File: src/packages/migrate/src/commands/MigrateDev.ts [hunk 2]
```diff
@@ -45,10 +45,9 @@ export class MigrateDev implements Command {
   }
 
   private static help = format(`
-${
-  process.platform === 'win32' ? '' : chalk.bold('🏋️  ')
-}Create a migration from changes in Prisma schema, apply it to the database, trigger generators (e.g. Prisma Client)
- 
+${process.platform === 'win32' ? '' : chalk.bold('🏋️  ')
+    }Create a migration from changes in Prisma schema, apply it to the database, trigger generators (e.g. Prisma Client)
```
Comment (tomhoule) [inline src/packages/migrate/src/commands/MigrateDev.ts:49]: prettier did that

File: src/packages/migrate/src/types.ts [hunk 3]
```diff
@@ -123,8 +123,8 @@ export namespace EngineResults {
     // Todo
   }
   export interface EvaluateDataLossOutput {
-    /// The migration steps that would be generated. If this is 0, we wouldn't generate a new migration, unless the `draft` option is passed.
-    migrationSteps: string[]
+    /// The number migration steps that would be generated. If this is 0, we wouldn't generate a new migration, unless the `draft` option is passed.
```
Comment (janpio) [inline src/packages/migrate/src/types.ts:126]:
```suggestion
    /// The number of migration steps that would be generated. If this is 0, we wouldn't generate a new migration, unless the `draft` option is passed.
```

**Comment by tomhoule** [comment]
https://github.com/prisma/prisma-engines/pull/2005 was just merged

**Comment by Jolg42** [comment]
We can merge after https://github.com/prisma/prisma/pull/7581

**Review by Jolg42 — APPROVED** [review]
Thanks!

**Pull request merged**